openjdk/openjdk18-image-stream.json

```diff
@@ -95,6 +95,59 @@
                     }
                 ]
             }
+        },
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "java",
+                "annotations": {
+                    "openshift.io/display-name": "Red Hat OpenJDK 8",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "1.4.11"
+                }
+            },
+            "labels": {
+                "xpaas": "1.4.11"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "8",
+                        "annotations": {
+                            "openshift.io/display-name": "Red Hat OpenJDK 8",
+                            "description": "Build and run Java applications using Maven and OpenJDK 8.",
+                            "iconClass": "icon-rh-openjdk",
+                            "tags": "builder,java,openjdk",
+                            "supports": "java:8,java",
+                            "sampleRepo": "https://github.com/jboss-openshift/openshift-quickstarts",
+                            "sampleContextDir": "undertow-servlet",
+                            "version": "8"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:latest"
```

Comment:
do we really want this pointing to "latest"?  it follows what we do for the SCL images, but the xpaas team doesn't seem to operate on the same model.  This configuration would mean anyone using the "java:8" IST is going to continuously pick up whatever is pushed to latest, for better or worse.  That's not what the previous(now hidden) imagestreams were doing.

Reply:
The downside to using a specific tag is that the image stream definition needs to be updated with every new version of the image.  If folks are referencing java:8, it doesn't really matter because as soon as the image stream definition is updated, they'll get the latest anyway.  If this is an issue, they would need to modify they're application to use a specific tag, e.g. `redhat-openjdk18-openshift:1.3`, so pointing this at latest doesn't really affect anything.